udata_gouvfr/views.py

```diff
@@ -72,6 +72,22 @@ def dataconnexions():
     return theme.render('dataconnexions.html', categories=categories)
 
 
+C3_PARTNERS = (
+    'institut-national-de-l-information-geographique-et-forestiere',
+    'meteo-france',
+    'etalab',
+    'ministere-de-l-ecologie-du-developpement-durable-et-de-l-energie',
+    'museum-national-dhistoire-naturelle',
+)
+
+
+@blueprint.route('/c3')
+def c3():
+    partners = Organization.objects(slug__in=C3_PARTNERS)
+    datasets = Dataset.objects(badges__kind=DATASET_BADGE_KINDS['c3'])
```

Comment:
You should use the [newly introduced](https://github.com/etalab/udata/pull/55/files#diff-69d191c77db7ac001b315c1625049ee7R54) `c3` var